examples/examples_go_test.go

```diff
@@ -66,7 +139,7 @@ func TestDockerfileGo(t *testing.T) {
 
 	opts := base.With(integration.ProgramTestOptions{
 		Dependencies: []string{
-			"github.com/pulumi/pulumi-docker/sdk/v4",
+			"github.com/pulumi/pulumi-docker/sdk/v4=../sdk",
```

Comment:
Yes and no. ProgramTest will copy the example program to `/tmp/$randomDir`, and the replace directive would not work from that location (`../sdk` => `/tmp/sdk`). 

Using this mode of `Dependencies` is just a little more explicit, and it will fail more loudly on major version updates if we forget to update it.